scvi/module/base/_base_module.py

```diff
@@ -74,6 +74,7 @@ class LossOutput:
     reconstruction_loss: LossRecord | None = None
     kl_local: LossRecord | None = None
     kl_global: LossRecord | None = None
+    classification: dict[str, Tensor] | None = field(default_factory=dict)
```

Comment:
Still not sure whether the keys in this argument should be strict or not. For now, I'm thinking that it should at least contain a classification loss, the ground truth labels, and either logits or predicted labels. logits are probably better since auroc does not accept predicted labels.

Reply:
my concern lies in the fact that all other arguments here are used in an explicit way, even `extra_metrics`, which just get logged. So I'm wondering if classification should also be a dataclass or more structured?

Reply:
I see your point, since all other args do get logged, but this isn't necessarily the case for `classification`. In that case, I feel like separating it out into a new dataclass would make sense

Reply:
alternatively, add labels and logits here just like kl_local and global